tern/extensions/cve_bin_tool/executor.py

```diff
@@ -23,7 +23,7 @@
 
 class CveBinTool(Executor):
     '''Execute cve-bin-tool'''
-    def execute(self, image_obj):
+    def execute(self, image_obj, redo=False):
```

Comment:
I would expect `passthrough.run_on_image` will also take the `redo` argument.

Reply:
@nishakm, I tried to debug the code but I could not see if we are loading data from the cache in case of cve-bin-tool. This tool is throwing an error to me and I think that is a tool bug so could not verify through a proper run too :(

Reply:
Oh oops! cve-bin-tool's extension doesn't use the cache. So your change here is correct. My apologies!